### ISSUE TITLE
How to add private content to the model v4

### ISSUE THREAD

**Issue by inspire-boy** [CompVis/stable-diffusion#174]
How to add private content to the model, such as adding your own image, what are the brief steps

**Comment by Forhek666** [comment]
python scripts/img2img.py --prompt "A fantasy landscape, trending on artstation" --init-img <path-to-img.jpg> --strength 0.8

**Comment by DasWookie** [comment]
> python scripts/img2img.py --prompt "A fantasy landscape, trending on artstation" --init-img <path-to-img.jpg> --strength 0.8

He is asking about how to further train the v4 model checkpoint sd-v1-4.ckpt.

**Comment by inspire-boy** [comment]
@DasWookie @Forhek666  yes,   thinks!  in this way, img i provide become diffrent. it lose original image.

**Comment by hopibel** [comment]
Use *low* strength (around 0.3) for img2img to preserve the input image better

**Comment by inspire-boy** [comment]
> 

Thank you very much. I will have a try.

**Comment by DasWookie** [comment]
Ahhh. Sorry, language barrier it appears. It seems that you AREN'T trying to add content for training to the model. So, you're just trying to preserve the look of an input image for img2img generation. In that case yeah, lower the strength value, or use masking to keep the area of the image you are trying to preserve.

**Comment by inspire-boy** [comment]
> 啊。抱歉，出现语言障碍。您似乎没有尝试向模型添加训练内容。因此，您只是想为 img2img 生成保留输入图像的外观。在这种情况下，是的，降低强度值，或使用遮罩来保留您尝试保留的图像区域。

Thank you. It's very interesting. It replaces PS to some extent

**Comment by inspire-boy** [comment]
> Ahhh. Sorry, language barrier it appears. It seems that you AREN'T trying to add content for training to the model. So, you're just trying to preserve the look of an input image for img2img generation. In that case yeah, lower the strength value, or use masking to keep the area of the image you are trying to preserve.

I have another problem. How to use prompt to accurately find a TV character, such as Tom cat. I find that I can't generate Tom. Use: "Tom cat playing in garden"  Or any other words
